### PR TITLE
keith/OFFAPPS-1084

### PR DESCRIPTION
**Pull request by keitheous**
OFFAPPS-1084 - Remove space on page load

## Description
Randomly, there are additional spaces below the app when loading different pages. Upon inspection, I found out that this is related and cause by this [issue](https://zendesk.atlassian.net/browse/OFFAPPS-1085). The buttons are squashed together in place, causing the html body to create additional space below the app.

## Task
- [ ] **Important**: Approve and merge this pr first before even looking at (https://github.com/zendesk/notification_app/pull/65)

## Implementation notes
I have decided to branch off this [pull request](https://github.com/zendesk/notification_app/pull/65) instead. In other words, this PR is a pre-requisite to the other PR.

I realised the app size isn't quite right when the app is loaded so i fixed that.

<details>
  <summary>App on page load - Before (left) & After (right)</summary>
screen shot 2018-10-30 at 5 12 05 pm
screen shot 2018-10-30 at 5 13 48 pm
</details>

## References
[JIRA ticket](https://zendesk.atlassian.net/browse/OFFAPPS-1084)
[Other PR](https://github.com/zendesk/notification_app/pull/65)

## CCs
@zendesk/apps-migration

## Risks
* [low] Resize app on page load.

### CODE REVIEW

File: src/javascript/app.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const appReady = new Promise((resolve) => {
 const app = {
   init: function () {
     storage('notifications', new Map())
+    appResize();
```
Comment (yu-tian113) [inline src/javascript/app.js]: Do you need to wait for `eClient.get('currentUser')` finish before resize?

Comment (ocke) [reply src/javascript/app.js]: I'm a bit puzzled how this `appResize` here fixes the problem. Later we do a `drawInbox` and then we resize again, shouldn't that be enough?

Comment (keitheous) [reply src/javascript/app.js]: You're right. I intend to close this PR and investigate further cause I realise doing this doesn't even fix the problem as it is unrelated.

**Comment by keitheous** [comment]
Closed for further investigation

**Comment by keitheous** [comment]
This issue is resolved in https://github.com/zendesk/notification_app/pull/71

**Pull request closed without merging**